doc/Architecture/arc-tec_x-road_technologies.md

```diff
@@ -74,71 +75,78 @@ See X-Road terms and abbreviations documentation \[[TA-TERMS](#Ref_TERMS)\].
-| Spring Boot 2                      | X                   |                    |                         |                                   |
-| Vue.js 2                           | X                   |                    |                         |                                   |
-| Npm 6                              | X                   |                    |                         |                                   |
-| Node 12                            | X                   |                    |                         |                                   |
-| Typescript                         | X                   |                    |                         |                                   |
-| OpenAPI 3                          | X                   |                    |                         |                                   |
-| Embedded Tomcat 9                  | X                   |                    |                         |                                   |
-| GNU Privacy Guard                  | X                   | X                  |                         |                                   |
+| **Technology**                     | **Security Server** | **Central Server** | **Configuration proxy** | **Operational Monitoring Daemon** |
+|------------------------------------|:-------------------:|:------------------:|:-----------------------:|:---------------------------------:|
+| Java 11                            |          X          |         X          |            X            |                 X                 |
+| C                                  |          X          |         X          |                         |                                   |
+| Logback                            |          X          |         X          |            X            |                 X                 |
+| Akka 2                             |          X          |         X          |            X            |                 X                 |
+| Jetty 9                            |  X\[[3](#Ref_3)\]   |  X\[[4](#Ref_4)\]  |                         |                                   |
+| Ubuntu 20.04                       |          X          |         X          |            X            |                 X                 |
+| Ubuntu 22.04                       |          X          |         X          |            X            |                 X                 |
+| Red Hat Enterprise Linux 7 (RHEL7) |          X          |                    |                         |                 X                 |
+| Red Hat Enterprise Linux 8 (RHEL8) |          X          |                    |                         |                 X                 |
+| PostgreSQL 10                      |          X          |         X          |                         |                 X                 |
```

Comment:
Should PostgreSQL 12 (Ubuntu 20.04) and PostgreSQL 14 (Ubuntu 22.04) be added? Also, PostgreSQL versions used by RHEL7 and RHEL8 should be checked.